### PR TITLE
Suggested edits to .mds

### PR DESCRIPTION
**Pull request by ljhwang**
Adds a few commas and attempts to clarify wording.  Please check to see if I inadvertently changed the intended meanings.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,11 +75,11 @@ road block if you would like to contribute some code.
 
 ## Acknowledgment of contributions
 While we are grateful for every contribution, there are also several official
-ways how your contribution will be acknowledged by the SOFTWARE_TEMPLATE community:
+ways your contribution will be acknowledged by the SOFTWARE_TEMPLATE community:
 - Every commit that was merged into the SOFTWARE_TEMPLATE repository will make you part of
   the growing group of SOFTWARE_TEMPLATE
   [contributors](https://github.com/geodynamics/software_template/graphs/contributors).
-- Mention any other kind of attribution your project will provide (e.g. newsletters, release papers, manual authorship).
+- Mention through any other kind of attribution the project provides (e.g. newsletters, release papers, manual authorship).
```
Comment (gassmoeller) [inline CONTRIBUTING.md:82]: I do not quite understand the `through`. What do you want to say?

**Review by gassmoeller — COMMENTED** [review]
Looks good except for the one sentence that I do not quite understand. Can be merged once that is clarified.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ljhwang) [reply CONTRIBUTING.md:82]: This was my attempt on interpreting the original sentence.  What was the intent?

Comment (gassmoeller) [reply CONTRIBUTING.md:82]: I meant the projects should mention here all other forms of attribution they will provide for their contributors, for example that their contributions will be announced in newsletters or they will be listed as authors.

**Review by gassmoeller — APPROVED** [review]
I forgot to merge this before. Ready.

**Pull request merged**